src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -140,4 +141,37 @@ case class Run(runId: String, pipeline: Pipeline, logger: WorkflowLogger) {
     val cancellationRequest: CancelOperationRequest = new CancelOperationRequest()
     pipeline.genomicsService.operations().cancel(runId, cancellationRequest).execute
   }
+
+  private def getEventList(op: Operation): Seq[ExecutionEventEntry] = {
+
+    // An execution event entry with only a "startTime"
+    case class HalfJesEvent(name: String, timestamp: DateTime)
```

Comment:
or perhaps the class name could be something more along those lines and the field name could stay the same